### PR TITLE
Refactoring router

### PR DESCRIPTION
**Pull request by vladimyr**
Whitelist routes using request flow, securing server with [`helmet`](https://npm.im/helmet) :tada:

### CODE REVIEW

**Comment by vladimyr** [comment]
@underscope This is rebased to latest develop and ready for merge :bowtie:

File: server/user/index.js [hunk 1]
```diff
@@ -3,12 +3,15 @@
 const ctrl = require('./user.controller');
 const model = require('./user.model');
 const router = require('express-promise-router')();
+const auth = require('passport').authenticate('jwt');
```
Comment (underscope) [inline server/user/index.js]: Order ☝️

File: server/router.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const express = require('express');
+const auth = require('passport').authenticate('jwt');
```
Comment (underscope) [inline server/router.js]: Order...

**Comment by underscope** [comment]
@vladimyr Please merge afterwards....

**Pull request merged**